gprofiler/main.py

```diff
@@ -507,6 +524,22 @@ def parse_cmd_args() -> configargparse.Namespace:
 
     _add_profilers_arguments(parser)
 
+    spark_options = parser.add_argument_group("Spark")
+
+    spark_options.add_argument(
+        "--spark-sample-period",
+        type=int,
+        default=60,
+        help="Time in seconds between each metric collection",
+    )
+
+    spark_options.add_argument(
```

Comment:
For use cases of `--collect-spark-metrics` but no profiling - let's add an explicit flag `--no-profiling` or `--profiling-mode=disabled` (uses the existing `--profiling-mode` flag), that is saner to use than passing `--no-php --no-java --no-dotnet` etc.

I think `--profiling-mode=disabled` makes most sense.